examples/speech-app/src/client/SpeechComponents.tsx

```diff
@@ -143,7 +167,15 @@ export const SpeechComponents: React.FC = () => {
           {translation && (
             <div>
               <h2 className="text-indigo-800">
-                4. Your speech translated to French
+                4. Your speech translated to {lang}
+                  <select id="language-select" value = {lang} onChange={(e)=>handleLanguage(e)}>
```

Comment:
Please derive the list of languages from a common constant to be shared by both the frontend and backend, under `src/common`.